kubernetes/opentelemetry-demo.yaml

```diff
@@ -7766,7 +7768,7 @@ spec:
             value: service.name=$(OTEL_SERVICE_NAME),service.instance.id=$(OTEL_K8S_POD_UID),service.namespace=opentelemetry-demo,k8s.namespace.name=$(OTEL_K8S_NAMESPACE),k8s.node.name=$(OTEL_K8S_NODE_NAME),k8s.pod.name=$(OTEL_K8S_POD_NAME)
           resources:
             limits:
-              memory: 200Mi
+              memory: 500Mi
```

Comment:
Did we need 500Mi?  Just curious since I had it at 350Mi.

Reply:
You are right, 350Mi is enough. I've updated it.